test/runtests.jl

```diff
@@ -10,11 +10,7 @@ if VERSION >= v"1.9.0"
     # Aqua.test_project_toml_formatting(Rasters)
 end
 
-# Only test SMAP locally for now, also RasterDataSources because CI dowloads keep breaking
```

Comment:
We do need to leave this code in ;)

Reply:
I didn’t look into the files, but it appears a second time further down, right? Should I still put it back?

Reply:
Oh right its duplicated, no worries then